src/connector/Cargo.toml

```diff
@@ -15,7 +15,7 @@ normal = ["workspace-hack"]
 
 [dependencies]
 anyhow = "1"
-apache-avro = { git = "https://github.com/risingwavelabs/avro", rev = "89c2c128de93586465a7ea85b0a1c1a53082bba2", features = [
+apache-avro = { git = "https://github.com/risingwavelabs/avro", branch = "risingwave-patched", features = [
```

Comment:
I guess we should prefer `rev` than `branch` as it's not stable.

Reply:
But any change would still be caught by `Cargo.lock` and trigger a `cargo-lock` review. Right?